### DISCUSSION TITLE
Map does not show in other language

### DISCUSSION THREAD

**Discussion by tt053626** [open-sdg/open-sdg#1473] [Q&A] [answered]
Our map only shows in English but does not work in other languages (no polygon, no data, zoom in wrong place). Everything else works just fine. Any clue?
Link to the site: https://beyersar.github.io/SDGTK/2-1-1/
Data repo: https://github.com/beyersar/DataTK

The Map shows correctly in English:
![en](https://user-images.githubusercontent.com/79691995/142200853-e3cf0eb9-e355-4309-9222-7f0119d55950.PNG)

But it does not show in German (but text, table etc. work in German):
![de](https://user-images.githubusercontent.com/79691995/142200871-7c8fd9b0-28f7-438b-b97d-b8346f82321e.PNG)

**Comment by WohnungNDS** [comment]
@tt053626 perhaps you can provide a link to your site, your site repo an your data repo.... This makes it easier to take a look at your configuration

**Reply by tt053626** [reply]
@WohnungNDS thanks for the hint. Just updated the link.

**Comment by WohnungNDS** [comment]
@tt053626 at a first glance everthing seems to be ok. Something is wrong in the English map too. Clicking the timeslieder the data is not shown anymore..... any Ideas @LucyGwilliamAdmin

**Comment by brockfanning** [comment, marked as answer]
@tt053626 That was a past bug, but it got fixed as of version 1.4.0. We've since released 1.5.0 and 1.6.0. I would recommend at least upgrading to 1.4.0, and then if possible to continue to 1.5.0 and then 1.6.0. Here are the upgrade instructions:

* 1.4.0: https://open-sdg.readthedocs.io/en/latest/upgrades/upgrading-1-4-0/
* 1.5.0: https://open-sdg.readthedocs.io/en/latest/upgrades/upgrading-1-5-0/
* 1.6.0: https://open-sdg.readthedocs.io/en/latest/upgrades/upgrading-1-6-0/

**Reply by tt053626** [reply]
Thank you very much!
